src/components/Menu/MenuItem/MenuItem.tsx

```diff
@@ -0,0 +1,75 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import IconCaretRight from "@elements/Icon/Generated/IconCaretRight";
+import IconCheck from "@elements/Icon/Generated/IconCheck";
+import { IconSize } from "@elements/Icon/IconSize";
+import { merge } from "@utilities/merge";
+import { ReactElement } from "react";
+import { MenuItemContent, MenuItemContentProps, MenuItemContentSize } from "./MenuItemContent";
+
+export enum MenuItemStyle {
+    Primary = "Primary",
+    Danger = "Danger",
+}
+
+export enum SelectionIndicatorIcon {
+    Check = "Check",
+    CaretRight = "CaretRight",
+}
+
+export type MenuItemProps = {
+    style?: MenuItemStyle;
+    size?: MenuItemContentSize;
+    disabled?: boolean;
+    active?: boolean;
+    selectionIndicator?: SelectionIndicatorIcon;
+} & Omit<MenuItemContentProps, "iconSize">;
+
+export const getStyle = (isDangerStyle: boolean, isSmall: boolean, disabled: boolean, active: boolean): string => {
```

Comment:
how about passing in an object in set default values?

Reply:
I'm not sure what you mean here, can discuss on Wednesday :)

Reply:
not necessary since we pass in all styles anyway